build/pluginctl/main.go

```diff
@@ -55,6 +59,16 @@ func pluginctl() error {
 }
 
 func getClient() (*model.Client4, error) {
+	socketPath := os.Getenv("MM_LOCALSOCKETPATH")
+	if socketPath == "" {
+		socketPath = defaultSocketPath
+	}
+
+	client, err := getUnixClient(socketPath)
+	if err == nil {
+		return client, nil
```

Comment:
Can we log an info message?
```suggestion
 		log.Printf("Connecting using local mode over %s", socketPath)
 		return client, nil  		
```